src/Sylius/Bundle/CoreBundle/DependencyInjection/Compiler/SyliusPriceHistoryLegacyAliasesPass.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+final class SyliusPriceHistoryLegacyAliasesPass implements CompilerPassInterface
+{
+    public const PLUGIN_TO_SYLIUS_PREFIXES_MAP = [
+        'Sylius\PriceHistoryPlugin\Application\Checker\ProductVariantLowestPriceDisplayCheckerInterface' => 'Sylius\Component\Core\Checker\ProductVariantLowestPriceDisplayCheckerInterface',
+        'Sylius\PriceHistoryPlugin\Application\CommandDispatcher' => 'Sylius\Bundle\CoreBundle\PriceHistory\CommandDispatcher',
+        'Sylius\PriceHistoryPlugin\Application\CommandHandler' => 'Sylius\Bundle\CoreBundle\PriceHistory\CommandHandler',
+        'Sylius\PriceHistoryPlugin\Application\Logger' => 'Sylius\Bundle\CoreBundle\PriceHistory\Logger',
+        'Sylius\PriceHistoryPlugin\Application\Processor' => 'Sylius\Bundle\CoreBundle\PriceHistory\Processor',
+        'Sylius\PriceHistoryPlugin\Application\Remover' => 'Sylius\Bundle\CoreBundle\PriceHistory\Remover',
+        'Sylius\PriceHistoryPlugin\Application\Templating' => 'Sylius\Bundle\CoreBundle\Templating',
+        'Sylius\PriceHistoryPlugin\Domain\Factory' => 'Sylius\Component\Core\Factory',
+        'Sylius\PriceHistoryPlugin\Domain\Model' => 'Sylius\Component\Core\Model',
+        'Sylius\PriceHistoryPlugin\Infrastructure\Cli\Command' => 'Sylius\Bundle\CoreBundle\PriceHistory\Cli\Command',
+        'Sylius\PriceHistoryPlugin\Infrastructure\Doctrine\ORM' => 'Sylius\Bundle\CoreBundle\Doctrine\ORM',
```

Comment:
A few more are left, basically, anything that's not under `CoreBundle\PriceHistory` needs to be more specific